### DISCUSSION TITLE
What happens when you insert a blank, unburned disk?

### DISCUSSION THREAD

**Discussion by ben-barnard** [automatic-ripping-machine/automatic-ripping-machine#745] [Q&A] [answered]
New to the software - putting everything together now. My big use case is wanting to churn through a ton of old disks, many of them burned, which I’d like to trash. They are mixed in with blank disks, which I’d like to keep. Anybody deal with this before? Does it just spit them back out? Does it build a tiny blank iso?

**Comment by 1337-server** [comment, marked as answer]
It should eject the disc and treat it as an empty drive

**Reply by ben-barnard** [reply]
Thank you!
